src/Setup.elm

```diff
@@ -0,0 +1,166 @@
+module Setup exposing (Model)
+
+import Browser
+import Element exposing (Element, el, px, rgb255, text)
+import Element.Background as Background
+import Element.Border as Border
+import Element.Font as Font
+import Element.Input as Input
+import Html
+
+
+type alias Model =
+    { startYear : Int
+    , endYear : Int
+    , einsteinFactor : Float
+    , districtFactor : Float
+    , playOffFactor : Float
+    , offSeasonFactor : Float
+    }
+
+
+type Msg
+    = StartYear Int
+    | EndYear Int
+    | EinsteinFactor Float
+    | DistrictFactor Float
+    | PlayOffFactor Float
+    | OffSeasonFactor Float
+    | Continue
+
+
+main : Program () Model Msg
+main =
+    Browser.sandbox
+        { init = init
+        , view = view
+        , update = update
+        }
+
+
+init : Model
+init =
+    { startYear = 2017
+    , endYear = 2019
+    , einsteinFactor = 1
+    , districtFactor = 1
+    , playOffFactor = 1
+    , offSeasonFactor = 1
+    }
+
+
+stringToFloat : String -> Float
+stringToFloat number =
+    let
+        maybeFloat =
+            String.toFloat number
+    in
+    case maybeFloat of
+        Nothing ->
+            0
+
+        Just value ->
+            value
+
+
+yearsInput : Int -> Input.Label Msg -> (Float -> Msg) -> Element Msg
+yearsInput value label onChange =
+    el [ Element.centerX, Element.width <| px 345 ] <|
+        Input.slider
+            [ Element.height (Element.px 20)
+            , Element.behindContent
+                (Element.el
+                    [ Element.width Element.fill
+                    , Element.height (Element.px 6)
+                    , Element.centerY
+                    , Background.color <| rgb255 0 0 200
+                    , Border.rounded 4
+                    , Element.centerX
+                    ]
+                    Element.none
+                )
+            ]
+            { onChange = onChange
+            , label = label
+            , min = 2007
+            , max = 2019
+            , step = Just 1
+            , value = toFloat value
+            , thumb =
+                Input.defaultThumb
+            }
+
+
+factorInput : Float -> String -> (String -> Msg) -> Element Msg
+factorInput value labelText sendMsg =
+    el [ Element.alignRight, Element.width <| px 317 ] <|
+        Input.text [ Element.width <| px 60, Font.size 20 ]
```

Comment:
You use many hardcoded pixel measurements, which is not really a good idea - the UI can't scale for different screen sizes.